sdk/program-entrypoint/src/lib.rs

```diff
@@ -279,6 +284,48 @@ macro_rules! custom_panic_default {
     };
 }
 
+/// This is an efficient implementation fo custom panic. It contains two syscalls and has a size
```

Comment:
typo

Reply:
Fixed in https://github.com/anza-xyz/agave/pull/4306/commits/365bdf56261a45888d2785a1c121a726154b3762